test/test_helper.exs

```diff
@@ -1 +1,13 @@
 ExUnit.start()
+
+defmodule EPA.TestHelpers do
+  @type error_message :: String.t
+  @type name :: String.t
+
+  @spec error_message([name]) :: error_message
+  def error_message(names) do
+    names
+    |> Enum.map(&"System.get_env(\"#{&1}\") is not set correctly")
+    |> Enum.join("\n")
+  end
+end
```

Comment:
Newline here, please. It can break unix tools. Such a dumb thing for editors to do :/